code/modules/client/preference_setup/loadout/lists/armament.dm

```diff
@@ -115,6 +115,17 @@
 	path = /obj/item/tool/sword/machete
```

Comment:
Remove the machete if you dont mind as well

Reply:
sure, gimme a sec